packages/plugin-dom-editable/src/DomEditable.ts

```diff
@@ -35,12 +36,23 @@ export class DomEditable<T extends DomEditableConfig = DomEditableConfig> extend
                     } else {
                         root = new VElement({ htmlTag: 'jw-editable' });
                         // Semantic elements are inline by default.
-                        // We need to guarantee it's a block so it can contain other blocks.
-                        root.attributes.style = 'display: block;';
+                        // We need to guarantee it's a block so it can contain
+                        // other blocks.
+                        let attributes = root.modifiers.get(Attributes);
+                        if (!attributes) {
```

Comment:
Again, `Modifiers` extends `Array` so `find` already exists. As for `get` I think that would be pretty confusing. Imagine checking if the character is bold with `char.modifiers.get(BoldFormat)` and it always returns something... Granted, you might want to do something like `char.modifiers.has(BoldFormat)` instead but then (1) you always have to do the work twice (look if it has it with `has`, then retrieve it with `get`) and (2) I believe it's a bit dangerous because you would not assume that `get` adds something new to the array so if you don't know you'll just use `get` and _get_ trouble later on.
So I'm not in favor ;-)

Reply:
Could we come up with a name other than `get` that would imply that it creates it if it's not there ?

Reply:
`getOrCreate` ? :trolleybus:

Reply:
I re-read your original reply and I actually disagree. You don't need `has` if you have `find`. Granted, it only works if `filter` also works, which I'm not sure, but you're trying.

Reply:
The way to know if a character is bold is to see if it has the `BoldFormat` class among its modifiers.
A logical way to know if a character has the `BoldFormat` class among its modifiers is to call `find(BoldFormat)` on its modifiers.
If `find(BoldFormat)` _always_ returns a `BoldFormat`, I'm misled to believe the character is always bold. So I would need to check that in a different way. That's where `has` enters the picture.

So I don't really understand why you say:
> You don't need `has` if you have `find`.

Reply:
You misunderstood what I said, or I didn't explain it properly. The idea would be to have:
- `find`: returns only if found (this is equivalent to your current implementation of `get`)
- `get`: calls `find` and if it does not find then create and append the modifier on the fly before returning
- `filter`like find but returns all matches